### PR TITLE
Update ASR1001-X.yaml

### PR DESCRIPTION
**Pull request by goebelmeier**
Refinements for ASR1001-X

### CODE REVIEW

**Comment by goebelmeier** [comment]
Oh, that's interesting. Should _all_ FRUs be modeled as a module? So far I understand that things like line cards are modeled as modules, but things like power supplies and fans are not.

**Comment by DanSheps** [comment]
All should as they are swappable so it makes more sense to model them as modules so that they can be swapped out easily

File: device-types/Cisco/ASR1001-X.yaml [hunk 1]
```diff
@@ -1,37 +1,102 @@
 ---
 manufacturer: Cisco
-model: ASR1001-X
-slug: asr1001-x
+model: ASR 1001-X
+slug: asr-1001-x
```
Comment (DanSheps) [inline device-types/Cisco/ASR1001-X.yaml:4]: We shouldn't change this

File: device-types/Cisco/ASR1001-X.yaml [hunk 2]
```diff
@@ -1,37 +1,102 @@
...
-    type: 10gbase-x-sfpp
+u_height: 1.0
+is_full_depth: false
+subdevice_role: ''
+airflow: front-to-rear
+comments: ''
 console-ports:
-  - name: con 0
-    type: rj-45
+- name: Console
```
Comment (DanSheps) [inline device-types/Cisco/ASR1001-X.yaml:12]: Per the bot, needs indentation.  Please fix all locations.

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Do you still need this? @goebelmeier

**Pull request closed without merging**